src/store/actions/load-words.js

```diff
@@ -0,0 +1,8 @@
+import { getCorrectWords } from '../../service/service';
+
+export const LOAD_GAME = 'LOAD_GAME';
+
+export const loadWords = () => ({
```

Comment:
load game vs load words - there is a name mismatch
also, earlier it was `initGame`, maybe it would be good to use one name only

Reply:
oh, sorry, didn't notice, I'll fix it. 
but in `Game.js` I cannot use one name only (I tried before, believe me), linter doesn't let me (no-shadow rule).

Reply:
oh I see
maybe second argument of `connect` is not that needed and you can call `dispatch` directly in lifecycle method, there is little gain from specifying `mapDispatchToProps`, it does not do anything meaningful

Reply:
👍 I'll look into it

Reply:
well, it works but:
- I think it's... a bit messy :/ plus I need to validate `dispatch` prop with PropTypes
- moreover it's described as an anti-pattern in documentation: https://react-redux.js.org/using-react-redux/connect-mapdispatch#can-i-call-storedispatch

Reply:
> it's described as an anti-pattern

I meant something different…
I don't want `store.dispatch`
I suggested not passing `mapDispatchToProps` to `connect` and then using `dispatch` from props, see https://react-redux.js.org/using-react-redux/connect-mapdispatch#default-dispatch-as-a-prop